vm-builder/files/Dockerfile.img

```diff
@@ -97,7 +108,7 @@ RUN set -e \
     && /neonvm/bin/id -g sshd > /dev/null 2>&1 || /neonvm/bin/addgroup sshd \
     && /neonvm/bin/id -u sshd > /dev/null 2>&1 || /neonvm/bin/adduser -D -H -G sshd -g 'sshd privsep' -s /neonvm/bin/nologin sshd
 
-FROM alpine:3.19 AS builder
+FROM vm-runtime AS builder
```

Comment:
This is a revert of a previous change, I think unintended?
(probably my bad, sorry!)